fpmsyncd/fpmsyncd.cpp

```diff
@@ -318,11 +318,6 @@ int main(int argc, char **argv)
         {
             cout << "Connection lost, reconnecting..." << endl;
         }
-        catch (const exception& e)
-        {
-            cout << "Exception \"" << e.what() << "\" had been thrown in daemon" << endl;
-            return 0;
```

Comment:
can we revert this change since protocol is now handled?

Reply:
This is needed for better future debugging. With the catch statement when there is an exception we gracefully exit with 0 status and issues go unnoticed.
In order for better debugging it is important to have core generated. Hence I removed this catch statement